buildSrc/src/main/kotlin/buildsrc/conventions/maven-publish.gradle.kts

```diff
@@ -4,30 +4,41 @@ plugins {
     id("buildsrc.conventions.base")
     `maven-publish`
     signing
+    // NOTE: external plugin version is specified in implementation dependency artifact of the project's build file
+    id("io.github.gradle-nexus.publish-plugin")
```

Comment:
The gradle-nexus-publish plugin is a little bit unusual, because it can only be applied to the root project

https://github.com/gradle-nexus/publish-plugin#applying-the-plugin

> The plugin must be applied to the root project

(I consider this a flaw of gradle-nexus-publish, it shouldn't have such a strict requirement. Plugins (including buildSrc convention plugins) should be applicable anywhere.)

To workaround this, I would either

* make a `nexus-publish.gradle.kts` buildSrc convention plugin, and put all the gradle-nexus-publish config in there. But... since gradle-nexus-publish can *only* be applied to the root project, this plugin isn't really reusable. 
* So in this instance I think it's okay to just applied & config gradle-nexus-publish in the root `build.gradle.kts`. It's not a lot of config.

Reply:
Although, thinking about it, what do you want to use the gradle-nexus-publish for? It's not necessary for publishing to Maven Central.

Reply:
The reason to use the plugin was to be able to close and release the artifact from a Github Action, not sure how else to do that.

Reply:
Ah okay. It's possible to do it without the gradle-nexus-publish plugin. 

First I'd try and get it working locally. If you re-enable the `MavenProjectLocal` repository are you able to publish to `./build/maven-project-local`? Does it contain all the checksums and signatures?

Reply:
I think so, I was able to deploy manually from my local machine: https://central.sonatype.dev/artifact/io.kweb/kweb-core/1.0.0/overview